mlflow/utils/databricks_utils.py

```diff
@@ -261,6 +261,21 @@ def get_job_type():
         return _get_context_tag("jobTaskType")
 
 
+@_use_repl_context_if_available("jobType")
+def get_job_type_info():
+    try:
+        return _get_context_tag("jobType")
+    except Exception:
+        return None
+
+
+def get_experiment_name_from_job_id():
```

Comment:
I think the job_id is better passed in as an argument instead of hidden in the implementation.
def get_experiment_name_from_job_id(job_id):
  return "job:/" + job_id if job_id else None

I see the current implementation try to catch a throw in get_job_id() call. It's not clear to me this should be done but I have not tried to look deep inside. What's the reason for it?

Reply:
I added the try catch to follow conventions on how the other ones are done. I didn't change the pattern. But I am happy to do so

Reply:
If you want to keep the pattern, let's call it get_job_experiment_name() to make it clearer. Also returning None is not particularly useful here. We need a non-empty experiment name nonetheless. It's probably better to not intercept an throw from get_job_id() call here, just let it pop up.